12_glob_lab/globlab.txt

```diff
@@ -4,26 +4,31 @@ https://www.tocode.co.il/bundles/unix1/lessons/globlab
 1. Type in a glob pattern to match:
 
   1. All files whose name starts with a lowercase letter
-  Answer: 
+  Answer: [a-z]*
 
+  Ofir>> for some reason the shell is not case sensitive so also upper case letters are matched, same for the next quesiton. 
+  
   2. All files whose name starts with an uppercase letter
-  Answer:
+  Answer: [A-Z]*
 
   3. All files whose name ends in an "a", "e", "i", "o" or "u"
-  Answer:
+  Answer: *[aeiou].*
 
   4. All files whose name DOES NOT end in an "a", "e", "i", "o" or "u"
-  Answer:
+  Answer: *[!aeiou].*
 
   5. All files with extension ".c" that are in a folder named "src"
-  Answer:
+  Answer: ls src/*.c
 
   6. All files with extension ".c" under any folder
-  Answer:
+  Answer: ls */*.c
```

Comment:
correct! there is also a[ hidden recursive glob](https://github.com/tocodeil/unix1-course-examples/issues/68) in bash :)